assume/world.py

```diff
@@ -174,6 +181,8 @@ def setup(
         save_frequency_hours: int = 24,
         bidding_params: dict = {},
         learning_config: LearningConfig = {},
+        episode: int = 1,
+        eval_episode: int = 1,
```

Comment:
I'd rather have these two as part of the kwargs or the learning config and not as additional items to the world itself..?